docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -29,182 +29,408 @@ tags:
+  Temporal Credits are used to pay your Temporal Cloud consumption, including Support charges. Purchasing Temporal Credits provides [volume pricing](#prepaid).
+
+A Temporal Credit is equivalent to &#36;1 USD.
+For example, a credit purchase of &#36;3,000 results in 3,000 Temporal Credits.
+The minimum purchase of Temporal Credits purchase is &#36;2,500.
+Each month, your total invoice draws down from your Temporal Credits balance.
+
+Once you run out of Temporal Credits, you'll be automatically invoiced for &#36;2,500 or your last month's Temporal Cloud invoice multiplied by 12, whichever is greater.
+
+### How to purchase Temporal Credits? {#purchasing-credits}
+
+You may purchase Temporal Credits by contacting our team at [sales@temporal.io](mailto:sales@temporal.io) or reaching out to your dedicated account manager.
+Temporal Cloud credits are also available for purchase available through AWS Marketplace.
+
+## Pay-as-you-go {#payg}
 
-Storage costs are measured in gigabyte-hours (GBh) and include charges for active Workflows, active storage, and the long-term, "retained" storage of Event Histories of closed Workflows.
-These are measured per Namespace.
+**How does pay-as-you-go pricing work?**
 
-_Active storage_ measures the amount of storage used by active Workflows.
+Pay-as-you-go uses consumption pricing.
+This section explains how you're billed each calendar month and gives examples.
 
-When the execution of a Workflow ends, Temporal Cloud stores Event History for a defined Retention Period, for historical use.
-This is _retained storage_.
-Typical uses include compliance, debugging, workload refresh, and business analytics.
-Both kinds of storage have fixed costs.
+### Action pricing {#payg-action-pricing}
 
-| **Storage** | **Cost per GBh (USD)** |
+You are only billed for the Actions you use.
+Actions are metered and billed on a calendar month for each Namespace.
+The pay-as-you-go rate is &#36;25 per million Actions per month.
+This rate applies across all Namespaces:
+
+| Actions | &#36;25 per million Actions |
+| ------- | --------------------------------|
+
+**Example**
+
+If you use fewer than one million Actions per month, your bill for Actions will be less than &#36;25 for that month.
+For example, if you use 800,000 Actions in a month, you're billed:
+
+```
+0.8M Actions ⨉ $25 Per Million Actions= $20
+```
+
+### Storage pricing {#payg-storage-pricing}
+
+You are billed for  your Namespace’s Active and Retained Storage each calendar month:
+
+| **Storage** | **Price per GBh (USD)** |
 | ----------- | ---------------------- |
 | Retained    | &#36;0.00042           |
 | Active      | &#36;0.042             |
 
-If you purchase Temporal Cloud credits (as [outlined earlier](#action)), active storage costs are tiered and measured in gigabyte-hours.
+**Example**
+
+If you have 720 GBhr of Active Storage and 3,600 GBhr of Retained Storage in one calendar month your bill will be:
```

Comment:
On line 141, you refer to gigabyte-hours as GBh, but here you are using GBhr.

Reply:
Should be GBh, without the r. Fixing.

Reply:
Universal check performed.